### DISCUSSION TITLE
Branch name restrictions rejecting Onboarding branch push

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#15622] [Request Help] [answered]
While I'm rolling out Renovate in the wider organization, I came across a GitLab project that has put restrictions on the branch names.

I know that I can overwrite the branch name via several properties, by which I might be able get the branch name in the right format, but how would I do that while there is no branch to configure that in? There is no onboarding PR, as its branches push was rejected for not adhering to the branch name standard.

How would I go about this? Would I have to skip the onboarding process and replay it manually, or are there other ways out?

**Comment by viceice** [comment, marked as answer]
fasted way will get manual onboard.

longer way: configure a repositories object for that repo with the different required settings on global config.
